dex/src/main/scala/com/wavesplatform/dex/error/MatcherError.scala

```diff
@@ -84,7 +84,7 @@ case class OrderBookBroken(theAssetPair: AssetPair)
       orderBook,
       commonEntity,
       broken,
-      e"The order book for ${Symbol("assetPair") -> theAssetPair} is unavailable, please contact with the administrator"
+      e"The order book for ${Symbol("assetPair") -> theAssetPair} has been deleted or does not exist"
```

Comment:
Here too. We need a new class, e.g. `OrderBookNotFound` with `orderBook, commonEntity, notFound` trio